src/PostgREST/Config.hs

```diff
@@ -88,18 +81,105 @@ corsPolicy req = case lookup "origin" headers of
 prettyVersion :: Text
 prettyVersion = intercalate "." $ map show $ versionBranch version
 
+-- | Default OS-specific path to config file
+defConf :: FilePath
+defConf =
+#ifdef mingw32_HOST_OS
+  "%PROGRAMDATA%\\postgrest\\postgrest.conf"
+#else
+  "/etc/postgrest.conf"
+#endif
+
 -- | Function to read and parse options from the command line
 readOptions :: IO AppConfig
-readOptions = customExecParser parserPrefs opts
-  where
-    opts = info (helper <*> argParser) $
-                    fullDesc
-                    <> progDesc (
-                    "PostgREST "
-                    <> toS prettyVersion
-                    <> " / create a REST API to an existing Postgres database"
-                    )
-    parserPrefs = prefs showHelpOnError
+readOptions = do
+  args <- customExecParser parserPrefs opts
+
+  when (caExample args) $ do
```

Comment:
i guess it's good enough and it's cool that it works each time you load the config.
So maybe have the binary just document the `-c` option and also mention the paths it tries to look for the config file, and maybe eventually a link to docs/sample